blog/2020-10-23-supervisor-update.md

```diff
@@ -0,0 +1,90 @@
+---
+author: Joakim Sørensen
+authorURL: https://github.com/ludeeus
+authorTwitter: ludeeus
+title: "Upcoming changes to add-ons"
+---
+
+## GitHub Action
+
+You can now use our [builder][marketplace] as a [GitHub action][github_action]! :tada:
+
+This is already in use in our [hassio-addons repository][addons], you can see an example on how we implemented it [here][builder-action].
+
+It can be used both to ensure that the add-on will still build with changes made to your repository, and to publish the images as part of a release workflow. How to use the action is documented in the [builder repository][builder-action].
+
+Here is an example on how you can use it:
+
+```yaml
+jobs:
+  build:
+    name: Test build
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout the repository
+      uses: actions/checkout@v2
+    - name: Test build
+      uses: home-assistant/builder@master
+      with:
+        args: |
+          --test \
+          --all \
+          --target /data
+```
+
+This example will run a test build on all supported architectures of the add-on.
+
+:::tip
+Your repository is mapped to `/data` in the action, so if you have your add-on files in subdirectories, you need to supply `--target /data/{directoryname}` as an argument to the builder action.
+:::
+
+## Documentation
+
+First off, our [API documentation][api_docs] has been moved to the developer documentation site. During this move it also got a style update to make it easier to navigate. Some of the endpoints are still missing some content. If you have not yet met your quota for [Hacktoberfest] maybe you want to contribute some more details to our API descriptions?
+
+## API Changes
+
+- Using the `/homeassistant/*` endpoints is deprecated, and will be removed later this year. You need to use `/core/*` instead.
+- Using `http://hassio/` is deprecated, and will be removed later this year. You need to use `http://supervisor/` instead.
+- Using `HASSIO_TOKEN` is deprecated, and will be removed later this year. You need to use `SUPERVISOR_TOKEN` instead.
+- Deleting snapshots with `POST` calling `/supervisor/snapshots/<slug>/remove` is deprecated, and will be removed later this year. You need to use the `DELETE` method when calling `/supervisor/snapshots/<slug>` instead.
+- Using `X-Hassio-Key` header as authentication method is deprecated, and will be removed later this year. You need use an authorization header with a `Bearer` token instead.
+
+The [API documentation][api_docs] have been updated to reflect these changes.
+
+## Add-on options
+
+The permissions of the `/data/options.json` file, were changed from `644` to `600`. If your add-on is running as non-root and you are reading this file, it will now give you permission issues.
```

Comment:
```suggestion
The permissions of the `/data/options.json` file, is changed from `644` to `600`. If your add-on is running as non-root and reading this file, it will now give you permission issues.
```